phil/s013-course-periods.md

```diff
@@ -81,6 +81,19 @@ in the export for BE
 ]
 ```
 
+# Learning Guide
```

Comment:
For the perf report and roster there aren't any student views so I don't think those need to change?

For the learning guide we are happy to do whatever makes most sense for the FE.  Y'all already know whether the user is a teacher or student right? So you could have a URL of "courses/23/guide" for both teacher and student but hit different endpoints under the covers.  The other way of thinking of it is that since you know whether it is a teacher or student you could hit the same endpoint and just interpret the results differently.